### DISCUSSION TITLE
AWS Simple Email V2

### DISCUSSION THREAD

**Discussion by vhmoura** [aws/aws-sdk-net#3169] [Q&A] [answered]
Hi,

I am trying to use AWS Simple Email v2 sandbox where both receiver and sender have their email address verified. However, I get an error saying "Email address is not verified. The following identities failed the check in region EU-WEST-1: x@x.com, y@y.com".

The code is pretty simple and it uses AWS guidelines:

var Client = new AmazonSimpleEmailServiceV2Client(
   AmazonAccessKey,
    AmazonSecretAccessKey,    
    new AmazonSimpleEmailServiceV2Config { RegionEndpoint = RegionEndpoint.GetBySystemName(AmazonRegion) }
    );

var response = await Client.SendEmailAsync(request);

Any one knows how to overcome this problem?

Thanks,
Vini

**Comment by dscpinheiro** [comment]
Do you still see that error message when using a different SDK (for example, the AWS CLI `send-email` - https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ses/send-email.html)? Our SDK doesn't throw that message so I assume it's coming from the service.

I did find this post mentioning the same error message: https://stackoverflow.com/a/37528929

**Comment by vhmoura** [comment]
Hi,

Using the portal console I am able to send from both emails, but not using
c# SDK (AWSSDK.SimpleEmailV2).

Thanks.

**Comment by dscpinheiro** [comment]
Would you be able to try it with the CLI as well? What I was interested in was if the result would be the different using the same `SendEmailRequest` (which you didn't include in the original question). The console uses JS but it's not straightforward to see which parameters are set.

**Comment by vhmoura** [comment, marked as answer]
Hi.I got it working by changing the region to eu-north-1 which doesn’t make any sense considering that it was setup in the eu-west-1.Any ideas why would it fail? I don’t have access to CLI to try I am afraid.Thanks ViniOn 1 Feb 2024, at 17:29, Daniel Pinheiro ***@***.***> wrote:﻿
Would you be able to try it with the CLI as well? What I was interested in was if the result would be the different using the same SendEmailRequest (which you didn't include in the original question). The console uses JS but it's not straightforward to see which parameters are set.

**Comment by dscpinheiro** [comment]
Glad you got it working, but yeah that seems odd. 

One thing I'd double check is that the e-mail setup was done in the same region you're calling the API - `AmazonRegion` in your example  (the [Amazon SES email sending errors documentation](https://docs.aws.amazon.com/ses/latest/dg/troubleshoot-error-messages.html) mentions `Amazon SES has endpoints in multiple AWS Regions, and email address verification status is separate for each AWS Region. You must complete the verification process for each sender in the AWS Regions you want to use`).

**Reply by ashishdhingra** [reply]
@vhmoura Please review the note in the last https://github.com/aws/aws-sdk-net/discussions/3169#discussioncomment-8338081 and confirm if the issue is resolved.

**Reply by vhmoura** [reply]
Hi. The issue wasn’t resolved. Only way to make it work is how I explained. 
cheers
